bindings/python/tests/test_pandas.py

```diff
@@ -115,3 +115,43 @@ def test_execute():
 
     assert out.equals(expected)
     con.close()
+
+def test_create_table_from_dataframe():
```

Comment:
I propose we merge with this test skipped, and it can be a task to unskip it when resolving #2487